pkg/trustedresources/verify_test.go

```diff
@@ -173,26 +168,41 @@ func TestVerifyTask_VerificationPolicy_Success(t *testing.T) {
 	}
 
 	tcs := []struct {
-		name   string
-		task   v1beta1.TaskObject
-		source string
-		signer signature.SignerVerifier
+		name                      string
+		task                      v1beta1.TaskObject
+		source                    string
+		signer                    signature.SignerVerifier
+		verificationNoMatchPolicy string
 	}{{
-		name:   "signed git source task passes verification",
-		task:   signedTask,
-		source: "git+https://github.com/tektoncd/catalog.git",
+		name:                      "signed git source task passes verification",
+		task:                      signedTask,
+		source:                    "git+https://github.com/tektoncd/catalog.git",
+		verificationNoMatchPolicy: config.FailNoMatchPolicy,
 	}, {
-		name:   "signed bundle source task passes verification",
-		task:   signedTask,
-		source: "gcr.io/tekton-releases/catalog/upstream/git-clone",
+		name:                      "signed bundle source task passes verification",
+		task:                      signedTask,
+		source:                    "gcr.io/tekton-releases/catalog/upstream/git-clone",
+		verificationNoMatchPolicy: config.FailNoMatchPolicy,
+	}, {
+		name:                      "signed task with sha384 key",
+		task:                      signedTask384,
+		source:                    "gcr.io/tekton-releases/catalog/upstream/sha384",
+		verificationNoMatchPolicy: config.FailNoMatchPolicy,
+	}, {
+		name:                      "ignore no match policy skips verification",
```

Comment:
Are there matching policies for this test case and the next one? If so, can that be clarified somehow? If not, that seems like a bug